x-pack/plugins/ml/server/models/data_recognizer/modules/apm_transaction/ml/apm_metrics.json

```diff
@@ -0,0 +1,53 @@
+{
+  "job_type": "anomaly_detector",
+  "groups": [
+    "apm"
+  ],
+  "description": "Detects anomalies in transaction duration, throughput and error percentage for metric data.",
+  "analysis_config": {
+    "bucket_span": "15m",
+    "summary_count_field_name" : "doc_count",
+    "detectors" : [
+      {
+        "detector_description" : "high duration by transaction type for an APM service",
+        "function" : "high_mean",
+        "field_name" : "transaction_duration",
+        "by_field_name" : "transaction.type",
+        "partition_field_name" : "service.name"
+      },
+      {
+        "detector_description" : "transactions per minute for an APM service",
+        "function" : "mean",
+        "field_name" : "transactions_per_min",
+        "by_field_name" : "transaction.type",
+        "partition_field_name" : "service.name"
+      },
+      {
+        "detector_description" : "percent failed for an APM service",
+        "function" : "high_mean",
+        "field_name" : "transaction_failure_percentage",
```

Comment:
Can we call this `failed_transaction_rate` to stay consistent with [existing terminology](https://www.elastic.co/guide/en/kibana/current/transactions.html)?

Reply:
I will leave the decision on this one to you @sqren . If you edit the `field_name` here, just make sure you also edit the name of the corresponding aggregation in the datafeed config `x-pack/plugins/ml/server/models/data_recognizer/modules/apm_transaction/ml/datafeed_apm_metrics.json`